downstream/modules/automation-hub/proc-configure-automation-hub-server-cli.adoc

```diff
@@ -42,7 +43,7 @@ The following `ansible.cfg` configuration file example shows how to configure mu
 server_list = automation_hub, my_org_hub
 
 [galaxy_server.automation_hub]
-url=https://cloud.redhat.com/api/automation-hub/api/galaxy/ <1> <2>
+url=https://console.redhat.com/api/automation-hub/api/galaxy/ <1> <2>
```

Comment:
shouldn't this be `https://console.redhat.com/api/automation-hub/content/published/` ?
AFAIK /api/automation-hub/api/galaxy is never correct

Reply:
@himdel Thank you for clarifying. It looks like the content currently published is correct: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/getting_started_with_automation_hub/configure-hub-primary#proc-configure-automation-hub-server-cli

I'm going to close this PR and the associated Jira ticket.

Reply:
Um... @dcdacosta there may have been a misunderstanding here....

The change from cloud -> console is correct.
I was referring to the rest of the URL being incorrect.

Reply:
IIUC the first part should say....

> Set the `url` option for each server name. You must include the API prefix (`/api/galaxy/` for PAH, `/api/automation-hub/` for c.r.c) in the server URL:

and the second part should be

```
[galaxy_server.automation_hub]
url=https://console.redhat.com/api/automation-hub/ <1> <2>
```

Reply:
Hi @himdel, The [galaxy_server.automation_hub] was previously changed to `https://console.redhat.com/api/automation-hub/content/published/` per a conversation with Anshul Behl here: https://redhat-internal.slack.com/archives/C035VBWU98D/p1695140973026119. 

The sentence "You must include the `api/galaxy/` subdirectory in the server URL:" has been commented out/removed.

Therefore, I believe the documentation as published is correct. See here: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/getting_started_with_automation_hub/configure-hub-primary#proc-configure-automation-hub-server-cli

Reply:
I see :)

I agreee the documentation as published is good :+1: 

(That said, https://github.com/dcdacosta/red-hat-ansible-automation-platform-documentation/blob/main/downstream/modules/automation-hub/proc-configure-automation-hub-server-cli.adoc is still wrong - and that's the `main` branch.)